test/unit/spec/appWidgets/input/textInputSpec.js

```diff
@@ -25,24 +25,23 @@ define(['common/runtime', 'widgets/appWidgets2/input/textInput', 'testUtil'], (
         };
     }
 
-    function startWidgetAndSetTextField(widget, container, inputText) {
-        widget.start({ node: container }).then(() => {
-            const inputElem = container.querySelector('input[data-element="input"]');
-            inputElem.value = inputText;
-            inputElem.dispatchEvent(new Event('change'));
-        });
+    async function startWidgetAndSetTextField(widget, node, inputText) {
```

Comment:
This test module got refactored a bit. Some of the test setups had bus messages leaking between tests. I'm not sure how, or why it was sporadic. Refactoring to not use the `done` mechanism, but instead wait on messages, fixed it up.